Kudu.Services/Functions/FunctionController.cs

```diff
@@ -80,7 +81,7 @@ public Task<HttpResponseMessage> CreateOrUpdateArm(string name, ArmEntry<Functio
             var tracer = _traceFactory.GetTracer();
             using (tracer.Step("FunctionsController.list()"))
             {
-                var functions = (await _manager.ListFunctionsConfigAsync()).Select(f => AddFunctionAppIdToEnvelope(f));
+                var functions = (await _manager.ListFunctionsConfigAsync(ArmUtils.IsArmRequest(Request)?new FunctionTestData():null)).Select(f => AddFunctionAppIdToEnvelope(f));
```

Comment:
Missing spaces around `?`. Please make sure to let VS auto format the code so it looks consistent everywhere.